docs/reference_guides/apps/grid_integration/Bidder.rst

```diff
@@ -13,6 +13,155 @@ uncertain price scenario has a corresponding power output. As shown in the figur
 each of these uncertain price and power output pairs formulates a segment in the
 bidding curves.
 
+Here we present a stochastic bidding model for a renewable integrated energy system (Wind generator + PEM).
```

Comment:
wind should be lowercase, make sure you define PEM on first use

Reply:
This is addressed.